package.json

```diff
@@ -10,23 +10,29 @@
         "deploy:polygon": "npx hardhat run scripts/deploy.js --network polygon",
         "deploy:mumbai": "npx hardhat run scripts/deploy.js --network mumbai",
         "verify:polygon": "npx hardhat verify --network polygon",
-        "verify:mumbai": "npx hardhat verify --network mumbai"
+        "verify:mumbai": "npx hardhat verify --network mumbai",
+        "lint": "solhint --max-warnings 0 'contracts/**/*.sol'",
+        "slither": "slither . --ignore-compile --filter-paths 'node_modules' --disable-color --print human-summary"
     },
     "author": "Underfitted Social Club",
     "license": "MIT",
     "devDependencies": {
-        "@nomiclabs/hardhat-ethers": "^2.0.2",
-        "@nomiclabs/hardhat-etherscan": "^2.1.7",
+        "@nomiclabs/hardhat-ethers": "^2.0.3",
+        "@nomiclabs/hardhat-etherscan": "^2.1.8",
         "@nomiclabs/hardhat-waffle": "^2.0.1",
+        "@openzeppelin/contracts": "^4.4.1",
```

Comment:
I'm not quite sure about this being moved to devDependencies - since this contains actual Solidity building blocks used by the application itself right? So this should still be in dependencies...

Reply:
I remove unused library